VuChunk[] vu_chunk_array
bool lidar_status

